specification/resources/resource-manager/readme.md

```diff
@@ -80,6 +80,14 @@ input-file:
 - Microsoft.Authorization/2015-01-01/locks.json
 ```
 
+### Tag: package-policy-2017-06

```

Comment:
Will you be having the api-version `2017-06-01` in future without `-preview` in api-version? In that case, I'd recommend tag to reflect the `-preview`.